lib/ddtrace.rb

```diff
@@ -39,12 +39,12 @@ def configuration
       @configuration ||= Configuration.new
     end
 
-    def configure(target = configuration)
-      target = Pin.get_from(target) unless target.is_a?(Configuration)
-
-      return target unless block_given?
-
-      yield(target)
+    def configure(target = configuration, opts = {})
+      if target.is_a?(Configuration)
+        yield(target)
```

Comment:
Do we have to worry if a block is given? Or is it always expected if you pass a configuration object that you will always pass a block?

Reply:
That's a fair point. I think `block_given ? yield(target) : target` would be better here.